assets/i18n/en_US.json

```diff
@@ -151,6 +151,7 @@
         "restartAppForChanges": "Restart the app to apply changes",
         "deleteKeystoreLabel": "Delete keystore",
         "deleteKeystoreHint": "Delete the keystore used to sign the app",
+        "deleteKeystoneDialogText": "Are you sure you want to delete the keystore used to sign patched applications?",
```

Comment:
Typo, you wrote keystoNe. In the 
_showDeleteKeystoneDialog function name as well.